.travis.yml

```diff
@@ -18,6 +18,29 @@ env:
   - GO111MODULE=on
   - GOPROXY=https://proxy.golang.org
 
+jobs:
+  include:
+    - name: "build"
+      script: go build ./...
+    - name: "lint"
+      install: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0
+      script: golangci-lint run --deadline=5m
+    - name: "coverage"
+      script:
+      - go test ./... -coverprofile=coverage1.txt
+      - go test ./impl/integration/ -coverprofile=coverage2.txt -coverpkg=./...
+      after_success: bash <(curl -s https://codecov.io/bash)
+    - name: "docker integration test"
+      install: skip
+      script: ./scripts/integration.sh
+    - stage: deploy
+      if: branch = master AND type != pull_request
```

Comment:
Is this equivalent to the previous behavior, or more restrictive? Maybe split into 2 PRs in the latter case?

Reply:
This is exactly equivalent to previous behavior. 
I don't know why Travis made it seem like it wasn't